Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "Feather"
+uuid = "becb17da-46f6-5d3c-ad1b-1c5fe96bc73c"
+version = "0.5.1"
+
+[deps]
+Arrow = "69666777-d1a9-59fb-9406-91d4454c9d45"
+CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+FlatBuffers = "53afe959-3a16-52fa-a8da-cf864710bae9"
+Mmap = "a63ad114-7e13-5084-954f-fe012c677804"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+
```

Comment:
It'd be good to have a compat section here too; at least for julia, but probably also for DataFrames (it just released a breaking release in 0.18)

Reply:
Done for julia and DataFrames.